cmd/bbolt/main.go

```diff
@@ -1234,13 +1232,16 @@ func (cmd *benchCommand) runWritesWithSource(db *bolt.DB, options *BenchOptions,
 				if err := b.Put(key, value); err != nil {
 					return err
 				}
+
+				results.AddCompletedOps(1)
 			}
 
+			results.ReportProgress()
```

Comment:
See my comment above.
```suggestion
```

Reply:
This is required to trigger reporting progress before waiting for the next tick. Without this, the last operation/s number would be wrong because there would not be enough operations on the last period to fill the whole second.